distribution/docs/src/main/resources/content/_transformers/xformers-intro.adoc

```diff
@@ -30,7 +30,7 @@
 
 Transformers transform data to and from various formats.
 Transformers are categorized by when they are invoked and used.
-The <<_types_of_transformers,existing types>> are <<_available_input_transformers,Input transformers>>, <<_available_metacard_transformers,Metacard transformers>>, and <<_available_query_response_transformers,Query Response transformers>>.
+The <<_types_of_transformers,existing types>> are <<{integrating-prefix}available_input_transformers,Input transformers>>, <<_available_metacard_transformers,Metacard transformers>>, and <<_available_query_response_transformers,Query Response transformers>>.
```

Comment:
❓ Should the other links in this link have prefixes?

Reply:
yes